tests/javascript/captionjs/spec/javascripts/caption-spec.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ * @package     Joomla
+ * @subpackage  JavaScript Tests
+ * @since       3.6
+ * @version     1.0.0
+ */
+
+define(['jquery', 'testsRoot/captionjs/spec/javascripts/caption-spec-setup', 'jasmineJquery'], function ($) {
+
+    describe('JCaption applied to single image', function () {
+        it('Should have caption as "Joomla Title 1" under image', function () {
+            expect($('.single p')).toHaveText('Joomla Title 1');
+        });
+    });
+
+    describe('JCaption applied for multiple images', function () {
+        it('Should have caption "Joomla Title 1" under image 1', function () {
+            expect($('.multiple p').first()).toHaveText('Joomla Title 1');
+        });
+
+        it('Should have caption as "Joomla Title 2" under image 2', function() {
+            expect($('.multiple p').last()).toHaveText('Joomla Title 2');
+        });
+    });
+
+    describe('JCaption with empty title attribute value', function () {
+        it('Should not have a <p> element inside the image container', function () {
+            expect($('.empty #empty-title').parent()).not.toContainElement('p');
```

Comment:
Why don't you use $('.empty').not.toContain.. ? Instead of using .empty class you could also make the identifier of all the divs unique (for example <div id="empty") or do you plan to use it more then once? :)